proposed/2021/DotnetNugetTrustedSigners.md

```diff
@@ -0,0 +1,196 @@
+# Title
+
+- Author Name [Kat Marchan (@zkat)](https://github.com/zkat)
+- Start Date 2021-03-04
+- GitHub Issue (GitHub Issue link)
+- GitHub PR (GitHub PR link)
+
+## Summary
+
+This specifies the syntax and behavior for the `dotnet.exe nuget trusted-signers` command.
```

Comment:
Should we remove the .exe terminology? Small nitpick, but this suggests Windows only. We removed this in docs a little ago after customers kept asking:

![image](https://user-images.githubusercontent.com/1288848/110481858-a1f7c800-80ad-11eb-9a62-b72fa9176f70.png)